packages/eventual-send/src/E.js

```diff
@@ -304,24 +304,69 @@ export default makeE;
 
 /**
  * The awaited return type of a function.
+ * For the eventual result of an E call, @see {EResult} or @see {ECallableReturn}
  *
  * @template {(...args: any[]) => any} T
  * @typedef {T extends (...args: any[]) => infer R ? Awaited<R> : never} EReturn
  */
 
 /**
- * @template {import('./types.js').Callable} T
+ * An eventual value where remotable objects are recursively mapped to Remote types
+ *
+ * @template T
+ * @typedef {Awaited<T>} EResult
```

Comment:
How does `Awaited<T>` differ from `EResult<T>`?

Reply:
Ah. The PR comment says

> Add an `EResult` type which in the future can represent the return type of an eventual send call mapped as necessary to avoid having the caller rely on the locality of the result.

which explains why it is not redundant to introduce something that starts identical. However, I did not understand the rest of that sentence. Rephrase?

Reply:
Thanks! Resolving.

Reply:
Actually, can you also explain this (briefly?) in the doc-comment here? Thx.